jlm/llvm/opt/ScalarEvolution.hpp

```diff
@@ -188,23 +219,54 @@ class SCEVChrecExpr final : public SCEV
     return Loop_;
   }
 
+  SCEV *
+  GetStartValue() const
+  {
+    return Operands_[0].get();
+  }
+
+  void
+  SetStartValue(const std::unique_ptr<SCEV> & initScev)
```

Comment:
The name `SetStartValue` is not getting across that we are pushing a start value to the front.